IPA.Loader/Config/Config.cs

```diff
@@ -148,7 +149,7 @@ internal static IConfigProvider GetProviderFor(string modName, ParameterInfo inf
             return GetProviderFor(modName, prefs);
         }
 
-        private static Dictionary<IConfigProvider, Action> linkedProviders =
+        private static Dictionary<IConfigProvider, Action> ProviderChangeDelegate =
```

Comment:
I would have this be lowerCamelCase given that its a private field, not a property or anything that is used outside of this class.

Reply:
Yup, I'll change this. Thanks